File: iosApp/iosApp/Presentation/Main/Home/HomeEvent.swift
//
//  HomeEvent.swift
//  ConCafe
//
//  Created by 홍희표 on 2026/03/06.
//

import Foundation

enum HomeEvent {
    case openExternalLink(url: String)
    case navigateToCast(id: String)
    case navigateToCafe(id: String)
}

File: composeApp/src/commonMain/kotlin/com/hhp227/concafe/presentation/main/home/HomeEvent.kt
package com.hhp227.concafe.presentation.main.home

sealed interface HomeEvent {
    data class NavigateToExternalLink(val title: String, val url: String) : HomeEvent
    data class NavigateToCast(val id: String) : HomeEvent
    data class NavigateToCafe(val id: String) : HomeEvent
    data object NavigateToSignIn : HomeEvent
}
